### PR TITLE
Small grammar fixes

### PR DESCRIPTION
**Pull request by Nurou**
> _NOTE_:
>
> - It's a good idea to open an issue first to discuss potential changes.
> - Please make sure that you are _NOT_ opening a PR to fix a potential security vulnerability. Instead, please follow the [Security guidelines](../Security.md) to disclose the issue to us confidentially.

## 🧢 Checklist

- [ ] Documentation
- [ ] Tests
- [ ] Ready to be merged

## 🎫 Affected issues

Please [scout and link issues](https://github.com/nextauthjs/next-auth/issues) that might be solved by this PR.

Fixes: INSERT_ISSUE_LINK_HERE

## 📌 Resources

- [Security guidelines](../Security.md)
- [Contributing guidelines](../CONTRIBUTING.md)
- [Code of conduct](../CODE_OF_CONDUCT.md)
- [Contributing to Open Source](https://kcd.im/pull-request)

### CODE REVIEW

File: docs/docs/providers/email.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ You can override any of the options to suit your own use case.
 
 ## Configuration
 
-1. NextAuth.js does not include `nodemailer` as a dependency, so you'll need to install it yourself if you want to use the Email Provider. Run `npm install nodemailer` or `yarn add nodemailer`.
+1. NextAuth.js does not include `nodemailer` as a dependency, you'll need to install it yourself if you want to use the Email Provider. Run `npm install nodemailer` or `yarn add nodemailer`.
```
Comment (rhinodavid) [inline docs/docs/providers/email.md:34]:
```suggestion
1. NextAuth.js does not include `nodemailer` as a dependency; you'll need to install it yourself if you want to use the Email Provider. Run `npm install nodemailer` or `yarn add nodemailer`.
```

File: docs/docs/providers/email.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ providers: [
 
 2.2 **Using a configuration object**
 
-In your `.env` file in the root of your project simply add the configuration object options individually:
+In your `.env` file is in the root of your project, simply add the configuration object options individually:
```
Comment (rhinodavid) [inline docs/docs/providers/email.md:64]: The previous version made more sense to me.

File: docs/docs/providers/email.md [hunk 3]
```diff
@@ -250,4 +250,4 @@ By default, NextAuth.js will normalize the email address. It treats values as ca
 
 :::warning
 Always make sure this returns a single e-mail address, even if multiple ones were passed in.
-:::
\ No newline at end of file
+:::
```
Comment (rhinodavid) [inline docs/docs/providers/email.md:253]: What's the change here?

**Comment by balazsorban44** [comment]
Thanks, but I don't agree with the proposed changes here. Appreciate the PR anyway!💚

If you are interested in contributing, check out these good first issues https://github.com/nextauthjs/next-auth/labels/hacktoberfest

**Pull request closed without merging**